### PR TITLE
kdeApplications: 20.04.3 -> 20.08.0

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Motivation for this change

Include the KDE frameworks update 5.71 -> 5.73

konsole, gwenview and okular tested and working

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by peterhoeg** [comment]
Cc @ttuegel

**Comment by peterhoeg** [comment]
@grahamcofborg build akonadi akonadi-calendar akonadi-contacts akonadi-import-wizard akonadi-mime akonadi-notes akonadi-search akonadiconsole akregator ark baloo-widgets bovo bomber calendarsupport dolphin dolphin-plugins dragon elisa eventviews ffmpegthumbs filelight granatier grantleetheme gwenview incidenceeditor k3b kaddressbook kalarm kalarmcal kapptemplate kate kbreakout kcachegrind kcalc kcalutils kcharselect kcolorchooser kdav kdebugsettings kdegraphics-mobipocket kdegraphics-thumbnailers kdenetwork-filesharing kdenlive kdepim-runtime kdepim-addons kdepim-apps-libs kdf kdialog keditbookmarks kfind kfloppy kgeography kget kgpg khelpcenter kidentitymanagement kig kimap kipi-plugins kitinerary kio-extras kldap kleopatra kmahjongg kmail kmail-account-wizard kmailtransport kmbox kmime kmix kmplot knotes kolf kolourpaint kompare konsole kontact kontactinterface konquest konqueror korganizer kpat kpimtextedit ksmtp kqtquickcharts kpkpass krdc krfb kruler kspaceduel ksudoku ksystemlog ktnef ktouch kwalletmanager libgravatar libkcddb libkdcraw libkdegames libkdepim libkexiv2 libkgapi libkipi libkleo libkmahjongg libkomparediff2 libksane libksieve mailcommon mailimporter marble mbox-importer messagelib minuet okular picmi pimcommon pim-data-exporter pim-sieve-editor print-manager rocs spectacle yakuake

**Comment by freezeboy** [comment]
Should be merged with #93840 as it contains many kdeApps

**Comment by peterhoeg** [comment]
Calligra is working again. The following packages do not:

`drawpile drawpile-server-headless i-score k9copy libsForQt5.libqtav libsForQt512.khotkeys libsForQt512.kpmcore virt-manager-qt zanshin`

Unless anybody feels super strongly about these, I'll merge this so we get it in for 20.08.

**Comment by peterhoeg** [comment]
@GrahamcOfBorg build drawpile drawpile-server-headless

**Comment by peterhoeg** [comment]
@GrahamcOfBorg build drawpile-server-headless

**Comment by FRidh** [comment]
Do the NixOS tests work?

**Comment by ttuegel** [comment]
`libsForQt5.libqtav` is known to be broken, it doesn't have a stable release upstream with the newer Qt version.

It could be that the newer `khotkeys` and `kpmcore` are not compatible with Qt 5.12; should we mark `libsForQt512.khotkeys` and `libsForQt512.kpmcore` broken?

**Comment by peterhoeg** [comment]
>   should we mark libsForQt512.khotkeys and libsForQt512.kpmcore broken?

Done.

**Comment by peterhoeg** [comment]
>   Do the NixOS tests work?

They *did* when I ran it over the weekend, but now *everything* needs to be rebuilt again, so it'll take a while....

**Comment by FRidh** [comment]
No worries, if they did then its fine. In this case I suggest merging this into master as soon as https://github.com/NixOS/nixpkgs/issues/96159 is resolved so we can evaluate it quicker.

File: pkgs/development/libraries/kpmcore/default.nix [hunk 1]
```diff
@@ -3,11 +3,8 @@
 , libatasmart, parted
 , utillinux }:
 
-let
+stdenv.mkDerivation rec {
   pname = "kpmcore";
```
Comment (jonringer) [inline pkgs/development/libraries/kpmcore/default.nix:7]: name was being used,
```
undefined variable 'name' at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/packet-spot-eval-3/pkgs/development/libraries/kpmcore/default.nix:11:58
```

**Comment by peterhoeg** [comment]
So this is good to go now. We should be able to squeeze 20.08.1 into the 20.08 release as it's scheduled for very early Sept.

**Pull request merged**